app/cleanup.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
 function cleanupOldFiles(generator, javaDir, testDir, resourceDir, webappDir, testJsDir) {
     if (generator.isJhipsterVersionLessThan('2.27.1')) {
         //removeFile and removeFolder methods should be called here for files and folders to cleanup
+        generator.removefolder('.tmp');
```

Comment:
Not required coz next release will be major so we dont expect people to use on existing projects